README.md

```diff
@@ -0,0 +1,529 @@
+
+To clone your fork locally:
+
+1. Browse to your fork's page (located at `github.com/your_user_name/growth-guide`).
+2. Click on the green **Code** button and select **Open with GitHub Desktop**.
+3. Click on **Choose…** and select the local folder where you want your fork to live.
+4. Click on **Clone** and when prompted hit **To contribute to the parent project**.
+5. Click on **Continue** and you're done. Good job! You've just cloned your fork. 👏
+
+### Syncing your fork
+
+GitHub Desktop has done is magic and now your personal fork should be synced with Memo Bank's version of the guide. This mean that you'll be able to update your fork every time someone else updates the main version of the guide.
+
+To update your fork:
+
+1. Open **GitHub Desktop** and navigate to your `growth-guide` repository.
+2. Click on the **Fetch origin** button and let GitHub Desktop do its magic for you.
+
+Note: if you've just forked our `growth-guide` repository and cloned it on your Mac, then your local clone should be up to date with Memo Bank's main version. Fetching Memo Bank's main repository is only useful if you haven't updated your fork in a long while.
+
+### Installing computer stuff on your Mac
+
+Now that the code of our guide lives on your computer, you need to install a few things on your Mac to run the guide locally.
+
+Here's what you'll need to install:
+
+1. Homebrew (a sort of tech App store).
+2. Ruby (the language Jekyll is written in).
+3. Jekyll (the engine that powers our site).
+
+The good news is: you only have to do these tedious steps once. So hang on.
+
+#### Installing Homebrew
+
+Homebrew is a tool that allows you to install other tools on your Mac, using the Terminal. Think of it as an App Store, but for developers.
+
+To install Homebrew:
+
+1. Open your Terminal (in `Applications` > `Utilities`). Double click on `Terminal.app`.
+2. Paste: `/bin/bash -c "$ (curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+3. Hit the **Enter** key on your keyboard.
+4. You'll be asked to confirm (hit **Enter** again) and type your macOS user password.
+
+⚠️ Nothing will show up on your screen when you'll start typing your password in the Terminal, that's normal. This is a security feature. Just type your password and hit **Enter** again when you're done.
+
+👉 [See how to troubleshoot common Homebrew erros](https://github.com/lewagon/setup/blob/master/macos.md#homebrew)
+
+Once you've installed Homebrew, paste the following lines in your terminal (all at once). It will install [Git](https://git-scm.com) and [GitHub command line interface](https://cli.github.com) on your Mac.
+
+```
+brew upgrade git || brew install git
+brew upgrade gh || brew install gh
+```
+
+You may quit your Terminal after that (`command` + `Q`).
+
+☝️ Note that “Terminal apps“ don't show up in your `Applications` folder. They're development tools, not real apps with a visual interface like Slack, iMovie, or Figma.
+
+#### Installing Ruby
```

Comment:
macOS comes with a default Ruby version. Did you really need to go through these steps? 🤔

Reply:
You're right about macOS, but Jekyll's documentation advises against using the default Ruby version that ships with macOS:

> To install Jekyll on macOS, you need a proper Ruby development environment. While macOS comes preinstalled with Ruby, we don’t recommend using that version to install Jekyll. This external article goes over the various reasons [why you shouldn’t use the system Ruby](https://www.moncefbelyamani.com/why-you-shouldn-t-use-the-system-ruby-to-install-gems-on-a-mac/).

— https://jekyllrb.com/docs/installation/macos/

Reply:
Interesting. Thanks for the link.